TransIT.DAL/Repositories/ImplementedRepositories/WorkTypeRepository.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using LinqKit;
+using Microsoft.EntityFrameworkCore;
+using TransIT.DAL.Models;
+using TransIT.DAL.Models.Entities;
+using TransIT.DAL.Repositories.InterfacesRepositories;
+
+namespace TransIT.DAL.Repositories.ImplementedRepositories
+{
+    public class WorkTypeRepository :BaseRepository<WorkType>,IWorkTypeRepository
+    {
+        public WorkTypeRepository(TransITDBContext context) : base(context)
+        {
+        }
+
+        public override Expression<Func<WorkType, bool>> MakeFilteringExpression(string keyword)
+        {
+            return part => EF.Functions.Like(part.Name, '%' + keyword + '%');
```

Comment:
It would be nice to add filtering not only by name, but also by `EstimatedCost` and `EstimatedTime`

Reply:
btw look at `PartRepository` and add more filtering if you can or ask someone to do that. 
Nice example is `IssueRepository`, look how its done there.